pages/datasets/_datasetId.vue

```diff
@@ -282,7 +287,7 @@ export default {
         split(' '),
         this.formatMetric,
         propOr(0, 'size')
-      )(this.datasetDetails)
+      )(this.datasetInfo)
```

Comment:
This is perhaps a better example of where I think ramda is being overused sometimes.  

Ramda is a great library, but I have no idea on first glance what this says.  

It _looks_ like datasetInfo.size is an array?  And then you are getting the first element of that array?  And then passing that to formatMetric, and then splitting the resulting string on `' '`?

IMHO the same thing could be accomplished in vanilla JS...